.github/scripts/parse_buildkit_output.py

```
@@ -0,0 +1,369 @@
+
+        multipliers = {
+            "B": 1,
+            "KB": 1024,
+            "MB": 1024**2,
+            "GB": 1024**3,
+            "TB": 1024**4,
+        }
+
+        return int(val * multipliers.get(unit, 1))
+
+    def generate_report(self, build_data: Dict[str, Any], image_name: str = None) -> str:
```

Ruff (0.14.5)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)